recipes/ensembl-vep/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
   md5: ad9059806b061ac11bb3e7d9f51ca30d
 
 build:
-  number: 0
+  number: 1
   skip: True # [osx]
```

Comment:
Ooops sorry I'm still learning my way around this magic. I just unset the ones that don't need to be bumped and fixes the one that was missing a build string. Let me know if anything else looks off.